internal/rest/middleware.go

```diff
@@ -24,25 +35,61 @@ func appendMiddlewares(mw ...middleware) func(http.Handler) http.Handler {
 // logger logs the request and response
 func logger(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.URL.Path != "/v2/healthz" { // reduce log spam
-			slog.Debug("received request", "method", r.Method, "path", r.URL.Path)
+		if slices.Contains(ignoredPaths, r.URL.Path) {
+			next.ServeHTTP(w, r)
+			return
 		}
+
+		slog.Debug("received request", "method", r.Method, "path", r.URL.Path)
 		next.ServeHTTP(w, r)
 	})
 }
 
+// requestDurationMetrics measures the duration of the request and records it for Prometheus
```

Comment:
so, is this the latency measurement for CM on request handling?
is it possible to add test for it?

Reply:
This isn't actual latency, it's the processing time of the request
The metric is unit tested, not yet integration tested as part of the rest interface, and won't be included in this PR